src/pages/Trade/components/TradeOrderTable/index.tsx

```diff
@@ -207,10 +208,17 @@ export default function OrderTable() {
     setIsPayInvalid(false)
   }, [maxPay, formRef, setPay, insufficientCKB])
 
+  const [firstToken, secondToken] = Order.pair
+
+  const payDecimal = useMemo(() => {
+    const sudt = SUDT_LIST.find(s => s.info?.symbol === firstToken)
+    const erc20 = ERC20_LIST.find(e => e.tokenName === firstToken)
+    return sudt?.info?.decimals ?? erc20?.decimals ?? DEFAULT_PAY_DECIMAL
```

Comment:
For the current token whitelist model, it may be more appropriate to throw an error when a token is found to be unavailable.